redash/query_runner/__init__.py

```diff
@@ -67,6 +67,22 @@ def configuration_schema(cls):
     def run_query(self, query):
         raise NotImplementedError()
 
+    def fetch_columns(self, columns):
+        column_names = []
+        duplicates_counter = 1
+        new_columns = []
+
+        for col in columns:
+            column_name = col[0]
+            if column_name in column_names:
+                column_name += " " + str(duplicates_counter)
```

Comment:
@arikfr I prefer to use table_name.column_name in this duplicated issues, what do you think about it?  Can I get the table name of the given column in some place? Also how can I debug in console what I'm working with? I couldn't did that with `print`.